src/main/webapp/WEB-INF/tags/admin/sessions/institutionPanel.tag

```diff
@@ -33,7 +33,7 @@
                     <th onclick="toggleSort(this,5,sortDate)"
```

Comment:
See if you can remove the `4` and `5` here also, and make `sortDate` the 2nd parameter?